bin/.travis/trusty/setup_ezplatform.sh

```diff
@@ -101,6 +101,6 @@ echo '> Change ownership of files inside docker container'
 docker-compose exec app sh -c 'chown -R www-data:www-data /var/www'
 
 echo '> Install data'
-docker-compose exec --user www-data app sh -c "php /scripts/wait_for_db.php; php bin/console ezplatform:install ${INSTALL_TYPE}"
+docker-compose exec --user www-data app sh -c "php /scripts/wait_for_db.php; php -d memory_limit='512M' bin/console ezplatform:install ${INSTALL_TYPE}"
```

Comment:
I've had the issue here. The installation of `product_list.yml` requires ~460MB of memory, but the default value on Travis machine was 256MB. I'm not sure if we should leave this command like this or find another way to deal with it?

Reply:
Yep, this is tricky - we're setting COMPOSER_MEMORY_LIMIT, but not php memory limit. IMHO we can change it to:
`php -d memory_limit=${COMPOSER_MEMORY_LIMIT} ...`, which gives us at least more control over this value.